03/app.js

```diff
@@ -43,40 +44,20 @@ books.getAuthor = function(isbn) {
 
 books.getTitle = function(isbn, lang) {
 
-    if(typeof this[lang] === undefined){
+    const title = this[isbn]['title'][lang]
```

Comment:
Warto też sprawdzić czy samo `isbn` istnieje - to już miałaś tj:
Dokładniej rzecz ujmując kod poniżej oznacza: Jeśli taki isbn nie istnieje to zwróć null, w przeciwnym razie rób dalsze operacje
```
if(typeof this[isbn] === 'undefined' {
     return null;
}
```